src/pages/NftCollectionInfo/index.tsx

```diff
@@ -0,0 +1,172 @@
+import type { AxiosResponse } from 'axios'
+import { useParams, useLocation, useHistory, Link } from 'react-router-dom'
+import { useQuery } from 'react-query'
+import Content from '../../components/Content'
+import Pagination from '../../components/Pagination'
+import NftHolderList from '../../components/NftHolderList'
+import NftCollectionOverview from '../../components/NftCollectionOverview'
+import NftCollectionTransfers from '../../components/NftCollectionTransfers'
+import { v2AxiosIns } from '../../service/http/fetcher'
+import i18n from '../../utils/i18n'
+import styles from './styles.module.scss'
+import NftCollectionInventory from '../../components/NftCollectionInventory'
+
+export interface InventoryRes {
+  data: Array<{
+    id: number
+    collection_id: number
+    icon_url: string | null
+    owner_id: number
+  }>
+  pagination: {
+    count: number
+    page: number
+    next: number | null
+    prev: number | null
+    last: number
+  }
+}
+
+export interface TransferListRes {
+  data: Array<{
+    id: number
+    from: string
+    to: string
+    item: {
+      id: number
+      collection_id: number
+      token_id: string
+      name: string | null
+      icon_url: string | null
+      owner_id: number
+      metadata_url: string | null
+      cell_id: number | null
+    }
+    transaction: {
+      tx_hash: string
+      block_number: number
+      block_timestamp: number
+    }
+  }>
+  pagination: {
+    count: number
+    page: number
+    next: number | null
+    prev: number | null
+    last: number
+  }
+}
+export interface HolderListRes {
+  data: Record<string, number>
+}
+
+const tabs = ['transfers', 'holders', 'inventory']
+const PAGE_SIZE = 50
+const NftCollectionInfo = () => {
+  const { id } = useParams<{ id: string }>()
+  const { search } = useLocation()
+  const history = useHistory()
+  const q = new URLSearchParams(search)
+  const page = q.get('page') ?? '1'
+  const tab = q.get('tab') ?? tabs[0]
+
+  const { isLoading: isTransferListLoading, data: transferListRes } = useQuery<AxiosResponse<TransferListRes>>(
+    ['nft-collection-transfer-list', id, page],
```

Comment:
`useQuery` first parameter is `The query key to use for this query.`
I don't find the use of `nft-collection-transfer-list`. And it's a const string that will not change.